src/plugins/inputs/otr.js

```diff
@@ -0,0 +1,16 @@
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "otr") {
+		return;
+	}
+	if (args.length === 0 || args[0] === "") {
```

Comment:
What I was pointing was that shift was called prior to splitting, you added that we are checking the 2nd element of list, so you proved it's safe.

> Don't you think we should leave it there just to be safe, until someone courageous edits this to make sure there is never an empty string there? (and doubles this courage with some shift() butchering)

There is no magic, if there is no edge case making it fail, let's not keep it :)

More globally, I think relying on internal API sane input is something good and makes code more  readable, I hate when same checks are done at each call level of the very same soft (I'm not speaking about 3rd-party libs).

PLC Even in case `arg[0] === ""`, which will not happen, code will behave the same here.